src/parser/demonhunter/havoc/modules/talents/BlindFury.js

```diff
@@ -57,9 +57,9 @@ class BlindFury extends Analyzer{
   suggestions(when) {
     when(this.suggestionThresholds)
       .addSuggestion((suggest, actual, recommended) => {
-        return suggest(<>Try to cast <SpellLink id={SPELLS.EYE_BEAM.id} /> at 30 to 50 fury. Having more than 50 fury at cast is considered a dps loss.</>)
+        return suggest(<>Cast <SpellLink id={SPELLS.EYE_BEAM.id} /> with 50 or less Fury to minimize Fury waste and maximize DPS.</>)
           .icon(SPELLS.BLIND_FURY_TALENT.icon)
-          .actual(`${actual} bad casts`)
+          .actual(<>{actual} bad <SpellLink id={SPELLS.EYE_BEAM.id} /> casts above 50 Fury. </>)
```

Comment:
I've never seen Fragments used here - does it work as intended? Can you verify? Otherwise switch to `${actual} bad Eye Beam casts above 50 Fury`

Reply:
Yup it works! Screenshot:
![image](https://user-images.githubusercontent.com/9600587/55111513-f5cf9900-50b0-11e9-82db-c4fcbb4b9aff.png)